src/config/index.ts

```diff
@@ -30,6 +30,10 @@ export const TEAM_OSPO_CHANNEL_ID = // #team-ospo
   process.env.TEAM_OSPO_CHANNEL_ID || 'G01F3FQ0T41';
 export const DISABLE_GITHUB_METRICS =
   process.env.DISABLE_GITHUB_METRICS == 'true';
+export const DRY_RUN =
+  process.env.DRY_RUN == 'true' || process.env.DRY_RUN == '1';
```

Comment:
I don't like supporting `yes` here as it's not your typical Boolean value. I think it's fine to put `process.env.DRY_RUN == 'true' || process.env.DRY_RUN == '1';` instead of complicating things. In Javascript, `true` and `false` are booleans, not `TRUE` or `FALSE` and so on so this seems fine to do